### DISCUSSION TITLE
无法登录

### DISCUSSION THREAD

**Discussion by wyj2046** [ZuodaoTech/everyone-can-use-english#875] [Q&A]
版本0.3.2
1、用github登录显示一直转圈
![Xnip2024-07-24_06-32-58](https://github.com/user-attachments/assets/518c3d0a-6cfb-4ce3-a55d-945812b1ae94)
2、用邮箱登录，点击发送验证码后显示network error
![Xnip2024-07-24_06-35-14](https://github.com/user-attachments/assets/7cb32c9f-dae5-4f1f-ab98-fa722f495186)

**Comment by an-lee** [comment]
点高级设置，把 API 改成 https://api.getenjoyapp.com
